src/js/check-theme.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-redeclare */
+/* eslint-disable quotes */
+/* eslint-disable no-var */
+/* eslint-disable indent */
+
+function getStyle(themeInfo) {
+  
+  if (themeInfo.colors.toolbar) 
+  {
+  
+    document.body.style.backgroundColor= themeInfo.colors.toolbar;
+  
+    document.getElementById('current-tab').style.color=themeInfo.colors.tab_background_text;
+   
+  const headers = document.getElementsByTagName('h3');
```

Comment:
We should add classes to the elements so we can style them by class instead of tag name.

Reply:
Could we add a new class name to each of them - e.g., `tabBackgroundText`?

Then we can stye them all with a single for loop:
```js
for (const elToStye of document.querySelectorAll(".tabBackgroundText")) {
  elToStyle.style.color =themeInfo.colors.tab_background_text;
}
```

Reply:
@groovecoder turns out I can be quite dim when it comes to optimizing code :D , this is definately better approach and reduces clutter. I'm updating the PR with these refactoring. Would love to improve with more suggestions like this.